### ISSUE TITLE
[Enhancement] Inject Other Properties Into Client Controllers

### ISSUE THREAD

**Issue by Rocky28447** [Sleitnick/AeroGameFramework#8]
It'd be nice if some other properties that were injected into client controllers. Properties like the LocalPlayer, the player's mouse, the CurrentCamera, and maybe some other things.

**Comment by Rocky28447** [comment]
Maybe like this? ;)
![image](https://user-images.githubusercontent.com/26193486/42303346-aa890050-7fee-11e8-81e7-0634a985c33c.png)

**Comment by Sleitnick** [comment]
I think injecting the Player object makes sense, but I'm not sure about Mouse and Camera. Specifically because the framework provides a custom Mouse user input object, and the Camera is not guaranteed to be the same (e.g. another script might switch the CurrentCamera). I'll update this once I make a decision.

**Comment by Rocky28447** [comment]
Yeah seems fair. It was kind of stupid of me to suggest injecting the camera. I realized the mistake with it immediately after posting and edited the post. Although, I didn't realize the framework came with a custom mouse input object. I'll be sure to check that one out!
